src/mslice/plotting/pyplot.py

```diff
@@ -2453,8 +2456,8 @@ def hlines(
 @_copy_docstring_and_deprecators(Axes.imshow)
 @set_category(CATEGORY_SLICE)
 def imshow(
-        X, cmap=None, norm=None, aspect=None, interpolation=None,
-        alpha=None, vmin=None, vmax=None, origin=None, extent=None, *,
+        X, cmap=None, norm=None, *, aspect=None, interpolation=None,
```

Comment:
Out of interest, what does the `*` parameter do/mean?

Reply:
From the [Python 3 documentation](https://docs.python.org/3/reference/compound_stmts.html#function-definitions):

Parameters after “*” or “*identifier” are keyword-only parameters and may only be passed by keyword arguments.